buf.yaml

```diff
@@ -0,0 +1,41 @@
+version: v2
+modules:
+  - path: internal/examples/pets/internal/proto
```

Comment:
nit: seems better to have `internal/proto` first -- it's the more useful one that someone is more likely to look at in this file, and it would keep them in alphabetical order.